curriculum/end2end/2.js

```diff
@@ -0,0 +1,14 @@
+const fs = require('fs');
+
+fs.readdir("/home", (err, files) => {
+    let users = [];
+    files.forEach(file => {
+          if(file!=="irishka2863"){
+                  users.push(file)
+                }
+        });
+  fs.writeFile(process.env.HOME+"/users.txt", users.join("\n"), (err) => {
+        if (err) throw err;
+        console.log('The file has been saved!');
+    })
```

Comment:
indent. In vim, type this:
gg=G

gg -> beginning of file.
= -> indent
G -> end of file.